dvc/output/base.py

```diff
@@ -58,6 +57,8 @@ class OutputBase(object):
     DoesNotExistError = OutputDoesNotExistError
     IsNotFileOrDirError = OutputIsNotFileOrDirError
 
+    sep = "/"
```

Comment:
is used anywhere? the same for the OutputLOCAL.

if yes, let's rename it to be consistent with const names

Reply:
It is still used in one place. This name is consistent with `os.sep` and `pathlib` internals.

Reply:
can you point me to the place please, just curious :) was not able to find it for whatever reason

Reply:
[Here] it is. It's in optimization part, so I wonder if changing that would hurt performance.